tests/chainerx_tests/unit_tests/routines_tests/test_creation.py

```diff
@@ -1174,3 +1176,55 @@ def test_copy(xp, shape, dtype, device, is_module):
         return xp.copy(a)
     else:
         return a.copy()
+
+
+@chainerx.testing.numpy_chainerx_array_equal()
+@pytest.mark.parametrize('shape,k', [
+    ((2,), -1),
+    ((2,), 0),
+    ((2,), 1),
+    ((3, 3), -1),
+    ((3, 3), 0),
+    ((3, 3), 1),
+    ((4, 3), -1),
+    ((4, 3), 0),
+    ((4, 3), 1),
+    ((4, 3), 5),
+    ((4, 3), -5),
+])
+@pytest.mark.parametrize_device(['native:0', 'cuda:0'])
+@chainerx.testing.parametrize_dtype_specifier('dtype_spec')
+def test_tri(xp, shape, k, dtype_spec, device):
+    if xp is numpy and isinstance(dtype_spec, chainerx.dtype):
+        dtype_spec = dtype_spec.name
+    out = xp.tri(*shape, k=k, dtype=dtype_spec)
+    if dtype_spec in (None, Unspecified):
+        out = dtype_utils.cast_if_numpy_array(xp, out, 'float32')
+    return out
```

Comment:
Can you add tests similar to `eye`, i.e. tests for default arguments and tests for the device argument.

Reply:
Done.